packages/css-data/src/property-parsers/transition.ts

```diff
@@ -0,0 +1,105 @@
+import * as csstree from "css-tree";
+import type {
+  InvalidValue,
+  LayersValue,
+  TupleValue,
+  TupleValueItem,
+  Unit,
+} from "@webstudio-is/css-engine";
+
+const cssTreeTryParseValue = (input: string) => {
+  try {
+    const ast = csstree.parse(input, { context: "value" });
+    return ast;
+  } catch {
+    return undefined;
+  }
+};
+
+export const parseTransition = (
```

Comment:
useless without tests no one will get what this function do

Reply:
definietely needs tests like other similar functions have

Reply:
Yeah, completely forgot adding them when opening for review. Will take a look at all the comments and the PR once again 👍